<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>强化模拟器</string>
  </property>
  <widget class="QLabel" name="pic">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>105</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>105</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>105</width>
     <height>105</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>请生成圣遗物</string>
   </property>
  </widget>
  <widget class="QLabel" name="relicName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Attri">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mainAttribute">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="levelup1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>升1级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="levelup4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>升4级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>主属性：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>51</y>
     <width>247</width>
     <height>447</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="chooseReilc">
    <item>
     <layout class="QHBoxLayout" name="choose">
      <item>
       <layout class="QVBoxLayout" name="attriOfRelic">
        <item>
         <widget class="QCheckBox" name="ckb1">
          <property name="text">
           <string>小生命值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb2">
          <property name="text">
           <string>百分比生命值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb3">
          <property name="text">
           <string>小攻击力</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb4">
          <property name="text">
           <string>百分比攻击力</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb5">
          <property name="text">
           <string>小防御力</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb6">
          <property name="text">
           <string>百分比防御力</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb7">
          <property name="text">
           <string>元素充能效率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb8">
          <property name="text">
           <string>元素精通</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb9">
          <property name="text">
           <string>暴击率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckb10">
          <property name="text">
           <string>暴击伤害</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="typeOfRelic">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>随机</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">relicTypeButton</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="flower">
          <property name="text">
           <string>生之花</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">relicTypeButton</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="feather">
          <property name="text">
           <string>死之羽</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">relicTypeButton</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hourglass">
          <property name="text">
           <string>时之沙</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">relicTypeButton</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bottle">
          <property name="text">
           <string>空之杯</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">relicTypeButton</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="helmet">
          <property name="text">
           <string>理之冠</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">relicTypeButton</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="createAsChoosed">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>生成圣遗物</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="randomCreate">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>随机生成</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="levelup">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>升满</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义词条</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义种类</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>--By Mr.AgNO3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="relicTypeButton"/>
 </buttongroups>
</ui>
